src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1023,3 +1024,29 @@ void AppHost::_UpdateTrayIcon()
         _trayIconData = nid;
     }
 }
+
+// Method Description:
+// - BODGY workaround for GH#9320. When the window moves, dismiss all the popups
+//   in the UI tree. Xaml Islands unfortunately doesn't do this for us, see
+//   microsoft/microsoft-ui-xaml#4554
+// Arguments:
+// - <none>
+// Return Value:
+// - <none>
+void AppHost::_WindowMoved()

```

Comment:
Does enumerating all the popups every time the window moves during a drag make drag performance slower?

Reply:
Not that I could perceive even in debug